src/models.py

```diff
@@ -71,6 +71,7 @@ def total_of_asset(cls, session: Session, user: User, asset: str, l2_network: st
 
 roles_users = Table(
     'roles_users',
+    BaseModel.metadata,
```

Comment:
this shouldnt be here, also will cause merge error

Reply:
removed the line above as its been implemented on another change.